bin/cron/export_mysql_database_to_redshift

```diff
@@ -29,6 +30,11 @@ def main
   end
   threads.each(&:join)
 
+  # The DMS tasks exported to staging (_import_) tables to avoid disrupting queries on the production Redshift tables
+  # during the lengthy table import process.  Rename the staging tables to the production table names to complete
+  # the import process
+  RedshiftImport.complete_table_import(Cdo::DMS.redshift_schemas_imported_from_database)
+
   ChatClient.message'data', "Completed export from Aurora MySQL database to Redshift."
 rescue StandardError => error
   ChatClient.message'data', "Error during export from Aurora MySQL database to Redshift #{error.message}", color: 'red'
```

Comment:
should there be a space between ChatClient.message and data?

Reply:
Yikes!  Rubocop probably didn't catch this because the file doesn't have a `.rb` extension.  Thank goodness I have `BenOCop` :)